cmd/libsnap-confine-private/utils-test.c

```diff
@@ -272,6 +272,55 @@ static void test_sc_array_size(void) {
 #endif
 }
 
+static void test_sc_is_path_clean(void) {
+    g_assert_true(sc_is_path_clean("/"));
+    g_assert_true(sc_is_path_clean("/.foo/bar"));
+    g_assert_true(sc_is_path_clean("/.foo/..bar"));
+    g_assert_true(sc_is_path_clean("/.foo/..bar/"));
```

Comment:
Would it be confusing that go's filepath.Clean would say no here? Perhaps we want to call it something other than `clean`.